kori/__main__.py

```diff
@@ -2,6 +2,7 @@
 from fastapi import FastAPI
 
 from kori.app.core.config import Settings
+from kori.app.routers.product import product_router
```

Comment:
I see that the linter has not been run.

Please fix it

EDIT: Its broken in many other locations, make sure to fix all files